quickstart/exampledata/config/prometheus/prometheus.yml

```diff
@@ -0,0 +1,32 @@
+# my global config
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
```

Comment:
The configured expose period in the quickstart `pipeline.yml` is set to `10`, so maybe we should set this scrape interval to something smaller than `10`?